src/Lykke.Service.Operations/Workflow/CashoutWorkflow.cs

```diff
@@ -273,28 +278,53 @@ public class CashoutWorkflow : OperationWorkflow
 
         private void SendToMe(CashoutMeInput input)
         {
-            var res = _exchangeOperationsServiceClient.ExchangeOperations.CashOutAsync(
-                new CashOutRequestModel
+            var policy = Policy
+                .Handle<ClientApiException>(exception =>
                 {
-                    ClientId = input.ClientId,
-                    Address = input.DestinationAddress,
-                    Amount = (double)input.Volume,
-                    AssetId = input.AssetId,
-                    TransactionId = input.OperationId.ToString(),
-                    FeeClientId = input.CashoutTargetClientId,
-                    FeeSize = input.FeeSize,
-                    FeeSizeType = input.FeeType == FeeType.Absolute ? FeeSizeType.ABSOLUTE : FeeSizeType.PERCENTAGE,
+                    _log.Warning("Retry on exception", context: exception.Message);
+                    return true;
                 })
-                .GetAwaiter().GetResult();
+                .Or<TaskCanceledException>(exception =>
+                {
+                    _log.Warning("Retry on exception", context: exception.Message);
+                    return true;
+                })
+                .OrResult<ExchangeOperationResult>(r =>
+                {
+                    _log.Info("Response from ME", r.ToJson());
+                    return r.Code == 500;
+                }) //ME runtime error
+                .WaitAndRetry(5, retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)));
 
-            if (!res.IsOk())
+            ExchangeOperationResult res = null;
+
+            policy.Execute(() =>
```

Comment:
Isn't it possible to do `var res = policy.Execute` instead of assigning variable in the closure?